icons/opensea.svg

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Before we review the SVG, could you make sure it [adheres to the requirements of sections 3, 4, and 5 of the Contributing Guidelines](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#3-optimize-the-icon)?

Running [SVGO](https://github.com/svg/svgo)([MG](https://jakearchibald.github.io/svgomg/)) on the SVG may suffice but be sure to check it manually as well :slightly_smiling_face: